internal/appsec/_testlib/mockspan.go

```diff
@@ -49,6 +49,6 @@ func (m *MockSpan) Context() ddtrace.SpanContext {
 	panic("unused")
 }
 
-func (m *MockSpan) StartChild(op string, opts ...ddtrace.StartSpanOption) ddtrace.Span {
+func (m *MockSpan) StartChild(op string, opts ...ddtrace.StartSpanOption) any {
```

Comment:
Is this `MockSpan` supposed to still exist even?

Reply:
Looks like it's still used, although I don't think it needs all of these methods anymore.